health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -76,23 +76,10 @@ async function fetchFileUrls(request: any, processedFileStoreIdForUSerOrFacility
-  const maxLengthRow = sheetData.reduce((maxRow: any, row: any) => {
-    return Object.keys(row).length > Object.keys(maxRow).length ? row : maxRow;
-  }, {});
-
-  // Extract headers from the keys of the row with the maximum number of keys
-  const originalHeaders = Object.keys(maxLengthRow);
-  const statusIndex = originalHeaders.indexOf('#status#');
-  // Insert 'errordetails' after '#status#' if found
-  if (statusIndex !== -1) {
-    originalHeaders.splice(statusIndex + 1, 0, '#errorDetails#');
-  }
+  const originalHeaders = type === config?.facility?.facilityType ? [config?.facility?.facilityCodeColumn, ...schema?.columns] : schema?.columns;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)